newspaper/extractors/articlebody_extractor.py

```diff
@@ -229,6 +229,7 @@ def is_tag_match(node, tag_dict):
         scores = []
         for tag in defines.ARTICLE_BODY_TAGS:
             if is_tag_match(node, tag):
+                assert isinstance(tag["score_boost"], int)
```

Comment:
not needed in my opinion, since tag is from the constant ARTICLE_BODY_TAGS, which is static defined, and should not be anything other than int

Reply:
`ARTICLE_BODY_TAGS` was defined as a dict with `str|int` keys, which meant it _could_ be a str from the typing perspective even if the actual data forbid it. I've rewritten `ARTICLE_BODY_TAGS` with an explicit TypedDict type to solve this.